lib/ex_webrtc/rtp_receiver/report_recorder.ex

```diff
@@ -39,10 +39,7 @@ defmodule ExWebRTC.RTPReceiver.ReportRecorder do
             jitter: 0.0,
             total_lost: 0
 
-  @doc """
-  Records incoming RTP Packet.
-  `time` parameter accepts output of `System.monotonic_time()` as a value.
-  """
+  # `time` parameter accepts output of `System.monotonic_time()` as a value.
```

Comment:
IMO the previous docs could stay

Reply:
I'd say to leave as a comment and in general use comments for developer information and docs for public stuff? The only useful thing I could think off was that the LSP would provide the doc info even when function is private, but it doesn't seem to be the case.